go.sum

```diff
@@ -0,0 +1,136 @@
+github.com/bep/clocks v0.5.0 h1:hhvKVGLPQWRVsBP/UB7ErrHYIO42gINVbvqxvYTPVps=
```

Comment:
For actively maintained libraries such as `github.com/google/go-github`, consider following their changelogs for any breaking changes that may affect your code after updates.